workflowhub_graph/merge.py

```diff
@@ -32,7 +35,7 @@ def merge_all_files(
 
     for i, fn in enumerate(filenames):
         with open(fn, "r") as f:
-            print(f"Processing {fn}, {i}/{len(filenames)}")
+            update_progress_bar(i + 1, len(filenames))
```

Comment:
Idle comment but could something like https://github.com/tqdm/tqdm work for this?